packages/vuetify/src/directives/ripple/__tests__/ripple.spec.ts

```diff
@@ -1,69 +1,34 @@
 // Libraries
-import Vue from 'vue'
+import { h, withDirectives, defineComponent, ref, createApp, nextTick, Ref } from 'vue'
 
 // Directives
 import Ripple from '../'
 
-// Utilities
-import {
-  mount,
-} from '@vue/test-utils'
-
 describe('ripple.ts', () => {
   it('Ripple with no value should render element with ripple enabled', () => {
-    const testComponent = Vue.component('test', {
-      directives: {
-        Ripple,
-      },
-      render (h) {
-        const data = {
-          directives: [{
-            name: 'ripple',
-          }],
-        }
-        return h('div', data)
-      },
-    })
-
-    const wrapper = mount(testComponent)
+    const Test = defineComponent(() => () => withDirectives(h('div', { class: 'a' }), [ [ Ripple, true ] ]))
+    const app = createApp()
+    const el = document.createElement('div')
+    app.mount(Test, el)
 
-    const div = wrapper.find('div')
-    expect(div.element['_ripple'].enabled).toBe(true)
+    expect(el.querySelector('.a')['_ripple'].enabled).toBe(true)
   })
 
-  it('Ripple should update element property reactively', () => {
-    const testComponent = Vue.component('test', {
-      directives: {
-        Ripple,
-      },
-      props: {
-        ripple: Boolean,
-        default: () => false,
-      },
-      render (h) {
-        const data = {
-          directives: [{
-            name: 'ripple',
-            value: this.ripple,
-          }],
-        }
-        return h('div', data)
-      },
-    })
-
-    const wrapper = mount(testComponent, {
-      propsData: {
-        ripple: true,
-      },
-    })
+  it('Ripple should update element property reactively', async () => {
+    const ripple = ref(true)
+    const Test = defineComponent((props: { ripple: Ref<boolean> }) => () => withDirectives(h('div', { class: 'a' }), [ [ Ripple, props.ripple.value ] ]))
```

Comment:
`props: reactive({ ripple: true ])`?

Reply:
we don't have `setProps` in Vue 3. `reactive` is great for multiple props, not for 1. I'm also unsure if it works.

Reply:
Unfortunately seems that it doesn't work https://codepen.io/jkarczm/pen/MWwgyYJ?editors=1010, however I believe that there's a better way to update component's prop than making prop a ref

And actually I think you don't need props here, this should work as well (didn't check tho):
```ts
const Test = defineComponent(() => () => withDirectives(
  h('div', { class: 'a' }), 
  [[Ripple, ripple.value]],
))
const app = createApp()
const el = document.createElement('div')
app.mount(Test, el)
```